src/App.js

```diff
@@ -44,27 +46,52 @@ function App() {
 
   return (
     <div className="App">
-      <header className="App-header">Sleep Tracker</header>
-      {savedGrid.map((day, i) => {
-        return <DayView key={i} row={day} />;
-      })}
-      <Day row={grid} mode={mode} changeRow={setGrid} />
-      <button onClick={clearGrid}>Clear</button>
-      <button onClick={saveGrid}>Save</button>
-      <div>
-        <input
-          onChange={handleModeChange}
-          id="sleep"
-          type="radio"
-          value="sleep"
-          name="mode"
-          defaultChecked
-        />
-        <label htmlFor="sleep">Sleep</label>
-        <input onChange={handleModeChange} id="awake" type="radio" value="awake" name="mode" />
-        <label htmlFor="awake">Awake</label>
-        <input onChange={handleModeChange} id="active" type="radio" value="active" name="mode" />
-        <label htmlFor="active">Active</label>
+      <div className="main-container">
+        <header className="section-container">
+          <h1> Sleep Tracker </h1>
+        </header>
+        <div className="days-container">
+          {savedGrid.map((day, i) => {
+            return <DayView key={i} row={day} />;
+          })}
+        </div>
+
+        <div>
+          <Day row={grid} mode={mode} changeRow={setGrid} />
+        </div>
+
+        <div className="section-container">
+          <input onChange={handleModeChange} id="active" type="radio" value="active" name="mode" />
+          <label htmlFor="active">Active</label>
+
+          <input
+            onChange={handleModeChange}
+            id="falling-asleep"
+            type="radio"
+            value="falling-asleep"
+            name="mode"
+          />
+          <label htmlFor="falling-asleep">Falling asleep</label>
+          <input
+            onChange={handleModeChange}
+            id="sleep"
+            type="radio"
+            value="sleep"
+            name="mode"
+            defaultChecked
+          />
+          <label htmlFor="sleep">Sleep</label>
+        </div>
+        <div className="section-container">
+          <DatePicker
+            className="date-picker"
+            selected={startDate}
+            onChange={(date) => setStartDate(date)}
+          />
+        </div>
+        <div className="section-container">
+          <Button action={saveGrid} name={'Save Sleep'} />
```

Comment:
In React, it's common to try to mimic "real HTML" as much as possible. And one of the basic fundamentals (yet the hardest to master) is creating good naming props. Compare to this one:

```jsx
<Button onClick={saveGrid}>
  Save Sleep
</Button>
```

This one it's more obvious what "action"  means `onClick`. Then for inner content, you can use `children` prop directly. How does that sound to you?